tests/Microsoft.Identity.Web.Test.Common/Mocks/QueryStringParser.cs

```diff
@@ -57,7 +57,11 @@ private static string UrlDecode(string message)
                 return message;
             }
 
+#if NET462 || NET472
```

Comment:
```suggestion
#if NETFRAMEWORK
```
I doubt they'll ever add it in...

Reply:
should possibly not even be called this though. we are starting to remove core from the naming since v6. So perhaps LegacyNetframework?

Reply:
`NETFRAMEWORK` is what the .NET team calls it. It's baked into all projects. No need to define a new pre-compilation flag.